src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partnerFieldText = `I am not married`;
+
+  if (isMarried && partnerName) {
+    partnerFieldText = sex === 'm'
+      ? `${partnerName} is my wife`
+      : `${partnerName} is my husband`;
+  }
```

Comment:
we can absolutely not use if

Reply:
yep, but in some cases if will be better than ternary operator